tests/integration/workflows/test_arxiv_merge.py

```diff
@@ -34,29 +34,49 @@
     start,
 )
 
+from factories.db.invenio_records import TestRecord
+
+RECORD_WITHOUT_CONFLICTS = {
+    'titles': [
+        {
+            'title': 'A title.'
+        },
+    ],
+    'document_type': ['article'],
+    '_collections': ['Literature']
+}
+
+RECORD_WITH_CONFLICTS = {
+    '$schema': 'http://schemas.stark-industries.com/schemas/records/avengers.json',
+    'titles': [
+        {
+            'title': 'A title.'
+        },
+    ],
+    'document_type': ['article'],
+    '_collections': ['Literature'],
+}
+
 
 @pytest.fixture
 def enable_merge_on_update(workflow_app):
-    with mock.patch.dict(workflow_app.config, {'FEATURE_FLAG_ENABLE_MERGER': True}):
+    with patch.dict(workflow_app.config, {'FEATURE_FLAG_ENABLE_MERGER': True}):
         yield
 
 
-def test_merge_with_disabled_merge_on_update_feature_flag(workflow_app, record_to_merge):
-    record_update = {
-        '$schema': 'http://localhost:5000/schemas/records/hep.json',
-        '_collections': ['Literature'],
-        'document_type': ['article'],
-        'titles': [
-            {'title': 'Jessica Jones'},
-            {'title': 'Luke Cage'},
-            {'title': 'Frank Castle'},
-        ],
-        'dois': [
-            {
-                'value': '10.1007/978-3-319-15001-7'
-            }
-        ],
-    }
+@patch(
+    'inspirehep.modules.records.api.fsopen',
+    mock_open(read_data='dummy body'),
+)
+def test_merge_with_disabled_merge_on_update_feature_flag(workflow_app):
```

Comment:
If I remove the mock then I'll have to mock either the request or file path which IMO is the same. The file location is inside the ``merge_record_with_documents.json`` and I want to keep it as pristine as possible. 

An ``invenio_files_rest`` factory will be very helpful in these cases.

Reply:
Forget about my comment here, I misunderstood.

Reply:
Now I looked better at the mock here. One of the main advantages of factories is that we do not have to use mocks anymore. But here with this new factory we always have to mock the `fsopen` call. A way to avoid this is to use the config `RECORDS_SKIP_FILES`, see:
https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/records/api.py#L93

This works well in your use case where you do not need files, but it does not solve the issue for the use case where files are required. If you have any idea, please share.